x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/querydsl/query/MatchQuery.java

```diff
@@ -65,15 +71,6 @@ public MatchQuery(Source source, String name, Object text, Map<String, String> o
         this.fuzziness = null;
     }
 
-    public MatchQuery(Source source, String name, Object text, Double boost, Fuzziness fuzziness) {
```

Comment:
This was coming from a previous attempt at writing fuzziness / boosting on the column operator - removing them as they will use the options when implemented